rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -26,8 +26,8 @@ elasticsearch_apt_packages:
   - openjdk-7-jre-headless
 
 elasticsearch_pip_packages:
-  - elasticsearch<2.1.0
-  - elasticsearch-curator<3.5.0
+  - elasticsearch>=2.0.0,<3.0.0
+  - elasticsearch-curator==4.0.4
```

Comment:
elasticsearch-curator didn't provide a sensible "version range" in the same way as elasticsearch did, so I went with pinning it to a specific version - happy to change.